docs/migration-from-4-to-5.md

```diff
@@ -0,0 +1,76 @@
+# Migration Guide from `v4` to `v5`
+
+The main change here is the upgrade from `stylelint` `v13` to `v14`, and therefore bumping all dependencies to `v14` compatible versions.
+
+## Installation
+
+NPM can be a bit funny about installs sometimes. If you are also using `eslint-config-skyscanner` it is recommended to reinstall `eslint-config-skyscanner` after this upgrade to ensure their shared dependencies are resolved correctly.
```

Comment:
Is it better to reinstall `eslint-config-skyscanner` or run `npm dedupe && npm prune`?

Reply:
In my experience when npm gets confused in this way, when it logically should not at all, then doing `npm install <library>` again is more foolproof. Here `npm ls prettier` was showing `prettier` was not installed at all, rather than duplicated. This was clearly not intended, the dep tree said it should be, but npm got itself in a knot - something I've seen it do more than once. Deleting `node_modules` did not help, but re-commanding the install did, and has in similar instances for me in the past.

Reply:
Perfect, thanks for sharing Dominic! 👏 
npm really struggles with this thing... 😅